nixos/modules/services/continuous-integration/buildbot/master.nix

```diff
@@ -225,16 +222,15 @@ in {
       };
 
       preStart = ''
-        ${pkgs.coreutils}/bin/mkdir -vp ${cfg.buildbotDir}
-        ${pkgs.coreutils}/bin/ln -sfv ${masterCfg} ${cfg.buildbotDir}/master.cfg
+        mkdir -vp ${cfg.buildbotDir}
+        ln -sfv ${masterCfg} ${cfg.buildbotDir}/master.cfg
+        rm -fv $cfg.buildbotDir}/buildbot.tac
         ${cfg.package}/bin/buildbot create-master ${cfg.buildbotDir}
+        echo "import sys" >> ${cfg.buildbotDir}/buildbot.tac
```

Comment:
> Or maybe there's a more native way to get it to spit logs to stdio?

Apparently launching through `twistd` is endorsed and supported. It seems like that might be best here:

```
twistd -n -l - -y ${cfg.buildbotDir}/buildbot.tac
```

instead of `buildbot start` should get us what we want with no .tac shenanigans!